runtime/src/main/java/org/corfudb/protocols/wireprotocol/StreamAddressRange.java

```diff
@@ -0,0 +1,24 @@
+package org.corfudb.protocols.wireprotocol;
+
+import lombok.Data;
+
+import java.util.UUID;
+
+/**
+ * This class represents a range of addresses for a stream.
+ *
+ * This is used to request the address map of a stream in
+ * a given boundary-- limits given by (end, start].
+ *
+ * Created by annym on 03/06/19
+ */
+@Data
```

Comment:
If your entity is immutable then please use `@Value`
`@Data` used for mutable entities